### PR TITLE
Add asio 1.28.2

### PR DESCRIPTION
**Pull request by tisonkun**
I wonder how can I test it locally ..

### CODE REVIEW

**Comment by tisonkun** [comment]
```

BcrValidationResult.FAILED: asio@1.28.2 is using an unstable source url: `https://github.com/chriskohlhoff/asio/archive/refs/tags/asio-1-28-0.zip`.
--
  | You should use a release archive URL in the format of `https://github.com/<ORGANIZATION>/<REPO>/releases/download/<version>/<name>.tar.gz` to ensure the archive checksum stability.
  | See https://blog.bazel.build/2023/02/15/github-archive-checksum.html for more context.


```

Asio upstream doesn't seems to have a `releases/download` artifacts. How can we do for this situation?

File: modules/asio/1.28.2/source.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "integrity": "sha256-XCrwfvc7QqLUjjTA7L9BzUDcgjvGgb9YM8EpOEmZyWM=",
```
[the remark this answers is not in the document]
Comment (fmeum) [reply modules/asio/1.28.2/source.json:2]: This is the Subresource Integrity Format: https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity#tools_for_generating_sri_hashes

[the remark this answers is not in the document]
Comment (tisonkun) [reply modules/asio/1.28.2/source.json:2]: Thank you! Updated.

**Comment by tisonkun** [comment]
CI gives green now. May you take another look @fmeum .

**Comment by fmeum** [comment]
@ltekieli Could you take a look as the current maintainer of the asio module?

**Review by fmeum — APPROVED** [review]
I'll assume that this is fine based on no response indicating the opposite (and it generally looks fine).

**Review by fmeum — APPROVED** [review]

**Pull request merged**